### ISSUE TITLE
Install newer redis pip package version in images

### ISSUE THREAD

**Issue by sapetnioc** [brainvisa/casa-distro#313]
Capsul v3 can use Redis' authentication/authorization system. But the `auth()` Python wrapping was introduced in redis-py v4.2.
To date it is possible to use it by upgrading via pip:  `bv pip install --upgrade --user redis`.
I think this issue will be resolved automatically when the next image is created.

**Comment by denisri** [comment]
I have built one yesterday. Is it OK ?

**Comment by sapetnioc** [comment]
No, I was wrong, Redis is currently installed with apt so the version is fixed (3.5.3). The future Ubuntu release (kinetic) will include v4.3.4. In the meantime, I think I will switch redis to an install with pip.
 
Moreover, Redis and Celery are only installed on dev image. We will have to move them to run image when we release Capsul v3. I do not see any interest to do it now.
